conference.js

```diff
@@ -1202,11 +1198,14 @@ export default {
             items[key] = param[1];
         }
 
-        this.e2eeKey = items.e2eekey;
+        if (typeof items.e2eekey !== undefined) {
+            APP.store.dispatch(setE2EEKey(items.e2eekey));
 
-        logger.debug(`New E2EE key: ${this.e2eeKey}`);
+            // Clean URL in browser history.
+            const cleanUrl = window.location.href.split('#')[0];
 
-        this._room.setE2EEKey(this.e2eeKey);
+            history.replaceState(history.state, document.title, cleanUrl);
```

Comment:
I don't have a specific example of a problem but I'm a little worried that this can get in conflict with our url params processing at some point. Any thoughts?

As I mentioned before if the whole thing is part of the code that already parses the URL I would be less worried about it :)

Also maybe we should remove the e2eekey only. WDYT?